cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/DefaultPluginConfigurer.java

```diff
@@ -83,8 +83,10 @@ public final <T> T usePlugin(String pluginType, String pluginName, String plugin
     try {
       Plugin plugin = addPlugin(pluginType, pluginName, pluginId, properties, selector);
       return pluginInstantiator.newInstance(plugin);
-    } catch (PluginNotExistsException | IOException e) {
-      LOG.error("Plugin name: {} with plugin type: {} cannot be added or instantiated", pluginName, pluginType, e);
+    } catch (PluginNotExistsException e) {
+      return null;
+    } catch (IOException e) {
+      LOG.error("Unexpected IOException", e);
```

Comment:
nit: it would be better to include the plugin name and type that had the problem, something like:

 `LOG.error("Unexpected error while instantiating {} plugin {}", pluginType, pluginName, e)`